CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -5,4 +5,13 @@
  * Collection of upgrade steps.
  */
 class CRM_ManualDirectDebit_Upgrader extends CRM_ManualDirectDebit_Upgrader_Base {
+
+  public function install() {
+    $this->executeSqlFile('sql/install.sql');
```

Comment:
rename install.sql to auto_install.sql and uninstall.sql to auto_uninstall.sql and this class https://github.com/compucorp/uk.co.compucorp.manualdirectdebit/blob/d99db9736df6e6b1760945bf86dbacaaa23f7c00/CRM/ManualDirectDebit/Upgrader/Base.php

will automatically handle the execution of these files, and then you can remove both install and uninstall methods from here.